shared/AppInsightsCore/src/JavaScriptSDK/HelperFuncs.ts

```diff
@@ -403,6 +403,11 @@ export function prependTransports(theTransports: TransportType[], newTransports:
 const strDisabledPropertyName: string = "Microsoft_ApplicationInsights_BypassAjaxInstrumentation";
 const strWithCredentials: string = "withCredentials";
 const strTimeout: string = "timeout";
+// used full url string here to avoid regex
+// or should we use regex here to include ...ai.config.1.cfg.json?dfkfhfhh as well
+const intExcludeTrackingEndpoints: string[] = [
+    "https://js.monitor.azure.com/scripts/b/ai.config.1.cfg.json"
```

Comment:
If we can avoid adding this would be better as this is an awful lot of uncompressible bytes and doesn't cover all domains